api/server/router/network/network_routes.go

```diff
@@ -104,11 +104,25 @@ func (n *networkRouter) postNetworkCreate(ctx context.Context, w http.ResponseWr
 		if _, ok := err.(libnetwork.ManagerRedirectError); !ok {
 			return err
 		}
+
+		var warning string
+		if _, ok := err.(libnetwork.NetworkNameError); ok {
```

Comment:
Thanks a lot for your review. It is my mistake so that what you said is totally reasonable.
I think it is better move this warning judgement before `ManagerRedirectError ` judgement. Now updated. PTAL, thanks.